Resources/Prototypes/Entities/Structures/Machines/artifact_analyzer.yml

```
@@ -157,3 +157,13 @@
       machine_parts: !type:Container
       entity_storage: !type:Container
       output_container: !type:Container
+
+# just a resprite
+- type: entity
+  parent: MachineArtifactCrusher
+  id: MachineBodyCrusher
```

```suggestion
  parent: MachineArtifactCrusher
  id: MachineArtifactCrusherXenoborg
```

Nitpick. These are basically same machines so better keep similiar ID so they're easier to search for.